src/test/java/edu/unc/lib/boxc/migration/cdm/SourceFilesCommandIT.java

```diff
@@ -117,6 +118,37 @@ public void generateBasicMatchDryRunTest() throws Exception {
         assertOutputMatches(".*Total Files in Project: +3.*");
```

Comment:
I think `generateBasicMatchDryRunTest` and `generateBasicMatchDryRunSummaryTest` are the exact same test now